cmd/manager/main.go

```diff
@@ -171,6 +179,40 @@ func main() {
 	}
 }
 
+func loadConfig() (*configuration.Registry, error) {
+	var configFilePath string
+	flag.StringVar(&configFilePath, "config", "", "path to the config file to read (if none is given, defaults will be used)")
```

Comment:
I don't think we need support for command line flags here

Reply:
This was also taken from the host operator. I had the same thought but assumed it was there for a reason so was hesitant to remove it. Any idea why it's in the host operator?

Reply:
nope, we don't use it so feel free to drop it from the host-operator as well ;-)

Reply:
Removed as part of 6d417df27d5ddb043b10285770c7a37c5b959286